app/lib/features/member/dialogs/show_member_info_drawer.dart

```diff
@@ -10,18 +11,30 @@ Future<void> showMemberInfoDrawer({
   bool isShowActions = true,
   Key? key = memberInfoDrawer,
 }) async {
-  await showModalBottomSheet(
-    showDragHandle: true,
-    enableDrag: true,
-    useSafeArea: true,
-    context: context,
-    isScrollControlled: true,
-    isDismissible: true,
-    builder: (context) => MemberInfoDrawer(
-      key: key,
-      roomId: roomId,
-      memberId: memberId,
-      isShowActions: isShowActions,
-    ),
-  );
+  context.isLargeScreen
+      ? await showDialog<void>(
+          context: context,
+          builder: (context) => AlertDialog(
+            content: MemberInfoDrawer(
+              key: key,
+              roomId: roomId,
+              memberId: memberId,
+              isShowActions: isShowActions,
+            ),
+          ),
+        )
+      : await showModalBottomSheet(
```

Comment:
what is the reasoning behind this? why didn't say we want user-dialogs to be center dialogs on larger screens ... is that fixing anything?

Reply:
I thought of it as UX-improvement as bottomsheet aren't usually well suited on larger screens, especially on large screenWidth having split views...